pyravendb/connection/requests_executor.py

```diff
@@ -140,13 +140,22 @@ def execute_with_node(self, chosen_node, raven_command, should_retry):
                         ("a certificate is required." if self._certificate is None
                          else name + " does not have permission to access it or is unknown.") +
                         response.request.method + " " + response.request.path_url)
+                if response.status_code == 410:
```

Comment:
What is this about?

Reply:
In c# this is case HttpStatusCode.Gone: // request not relevant for the chosen node - the database has been moved to a different one

Reply:
Okay